blocks/loader.ts

```diff
@@ -192,11 +192,16 @@ const wrapLoader = ({
 
         ctx?.monitoring?.tracer?.startActiveSpan?.("object-hash", (span) => {
           try {
-            const hashedProps = hash(props);
-            span.setAttribute(
-              "hash_size_bytes",
-              hashedProps.length*2,
-            );
+            const hashedProps = hash(props, {
+              ignoreUnknown: true,
+              respectType: false,
+              respectFunctionProperties: false,
+              algorithm: "md5",
+            });
+            span.setAttributes({
+              hash_size_bytes: hashedProps.length * 2,
+              props_length: JSON.stringify(props).length,
+            });
             url.searchParams.set("props", hashedProps);
```

Comment:
could this not run if otel is not active? if so, this is a bug. it should always run set("props").

Reply:
no, it will always run. Even if otel disabled